pkg/localstore/store_lock.go

```diff
@@ -0,0 +1,41 @@
+package localstore
+
+import "resenje.org/multex"
+
+const (
+	Upload   string = "upload"
```

Comment:
I do not thing that these constants need to be exported as they are only used as internal lock keys. Because of that, also, a more explanatory name would be good, like `lockKeyUpload` or something similar. When type parametrized multex version could be used, the custom type could be used for clarity instead naming. Just a suggestion.